README.md

```diff
@@ -8,6 +8,27 @@ The sources of the Cyphal specification and other related documents are containe
 When cloning this repository, don't forget to initialize the Git submodules:
 `git submodule update --init --recursive`.
 
+## Governence
+
+The Cyphal specification is community-managed and completely open. Anyone can propose changes but only Cyphal maintainers may commit changes to this repository where the contents of the HEAD revision of the primary branch constitutes the latest version of the specification in-effect.
+
+### Request For Comments (RFC) Process
+
+Changes to the specification shall use the following, community-driven RFC process:
+
+1. An RFC is posted on the [OpenCyphal forum's specification topic](https://forum.opencyphal.org/c/dev/spec/15) with the details of the proposed change. The title of this post must start with "RFC: " to indicate it is the start of a proposed change to the specification.
+2. An OpenCyphal maintainer must sponsor new RFCs within 10 days of the posting or they expire.
+    - Expired RFCs can be resubmitted after a three month cooling-off period.
+3. Once a maintainer has sponsored an RFC it remains open as long as there are active discussions.
+   - After 10 days of no additional comments or after the sponsoring Maintainer deems the discussion is complete the RFC is ready to be either accepted or rejected.
+   - An RFC shall remain open for comments for at least 10 days from the date it was sponsored regardless of the sponsoring Maintainer's opinion.
+   - Any maintainer can remove any open RFC at any time, without regard to any other rule, if it or the subsequent discussion violates the forum's community standards.
+   - An open RFC can be removed from the forum, without regard to any other rule, if two or more maintainers agree to remove it.
+4. Once an RFC is ready for a final disposition at least one maintainer, _other than the sponsoring maintainer_, and the majority of active maintainers shall determine if the RFC is accepted or rejected.
+    - Rejected RFCs can be resubmitted after a 3-month cooling-off period.
+5. After an RFC is accepted on the forum the change can be made to the specification in this repository. The Pull Request must reference the forum discussion and at least one maintainer must agree that the change is correct as specified by the text of the approved RFC.
+    - If an RFC is not reified by a change to the Specification within three months of approval then it expires. Expired RFCs can be resubmitted after a 3-month cooling off period.
```

Comment:
This expiration clause effectively means that if a proposal is not implemented in 3 months then one would have to wait extra three months before it is reintroduced.

Reply:
Well, by "implemented" we mean the specification is updated, right?

I'm trying avoid having a backlog of approved RFCs which would mean you need to know the current specification + the set of approved but not implemented RFCs to know the current state of the standard. My reasoning is that if the community is indifferent but willing to approve an RFC all the same _someone_ has to want it bad enough to make the change. If nobody does after the three months then nobody really cared enough and it probably should die.

The three-month thing in all of these steps is to avoid malcontents from abusing the RFC process to badger the community into accepting a change they repeatedly reject.

I'm open to modifying this, however. What would you propose?

Reply:
I would replace the three-month period with a "reasonable timeframe".